org.eclipse.jdt.debug.tests/tests/org/eclipse/jdt/debug/tests/AbstractDebugTest.java

```diff
@@ -1536,13 +1539,14 @@ protected IJavaThread launchToLineBreakpoint(String mainTypeName, ILineBreakpoin
 	 */
 	protected IJavaThread launchToLineBreakpoint(ILaunchConfiguration config, ILineBreakpoint bp, boolean register) throws Exception {
 		DebugEventWaiter waiter= new DebugElementKindEventDetailWaiter(DebugEvent.SUSPEND, IJavaThread.class, DebugEvent.BREAKPOINT);
-		waiter.setTimeout(DEFAULT_TIMEOUT);
+		waiter.setTimeout(30000);
```

Comment:
Why is this changed from `DEFAULT_TIMEOUT` to `30000`? Please restore. This will affect every other test too.

Reply:
Restored to previous.